psignifit/tools.py

```diff
@@ -79,14 +80,16 @@ def psychometric_with_eta(stimulus_level, threshold, width, gamma, lambda_,
     if random_state is None:
         random_state = np.random.default_rng()
 
-    sigmoid = sigmoid_by_name(sigmoid_name)
+    if type(sigmoid) is str:
```

Comment:
`if isinstance(sigmoid, str)` -> subclasses are welcome

Reply:
fixed in d50061ba0ab6975fa4ffe40966609e2bfa83cee2